cellbase-app/app/docker/cellbase-app/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM cellbase
+
+# Build args
+ARG TOMCAT_VERSION="8.5.38"
+ARG VERSION=""
+ARG VCS_URL=""
+ARG VCS_REF=""
+ARG BUILD_DATE=""
+
+# Metadata
+LABEL org.label-schema.vendor="OpenCB" \
+      org.label-schema.url="http://docs.opencb.org/" \
+      org.label-schema.name="cellbase" \
+      org.label-schema.description="An Open Computational Genomics Analysis platform for big data processing and analysis in genomics" \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.vcs-url=${VCS_URL} \
+      org.label-schema.vcs-ref=${VCS_REF} \
+      org.label-schema.build-date=${BUILD_DATE} \
+      org.label-schema.docker.schema-version="1.0"
+
+# Install local dependencies
+RUN apt-get update && apt-get install -y wget tar \
+  && rm -rf /var/lib/apt/lists/* \
+  && apt-get clean
+
+# Download and install Tomcat
+RUN wget --quiet --no-cookies http://www-eu.apache.org/dist/tomcat/tomcat-8/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz -O /tmp/tomcat.tgz && \
```

Comment:
Same as in https://github.com/opencb/opencga/pull/1216 , the URL used to download Tomcat will fail as soon as a new version is deployed. We should use the archive URL, or download from the repositories, with `apt-get`

Reply:
done.